fastlane/Fastfile

```diff
@@ -360,7 +360,7 @@ end
 
 desc 'Runs ui tests in Debug config'
 lane :test_ui do |options|
-  next unless is_check_required(sources: sources_matrix[:ui])
+  # next unless is_check_required(sources: sources_matrix[:ui])
```

Comment:
Why is this commented out?

Reply:
To test Snapshot tests on CI (that's why I missed snapshots in the first PR). I will revert it after the test.